CesiumGltf/include/CesiumGltf/PropertyView.h

```diff
@@ -1435,6 +1678,67 @@ class PropertyView<PropertyArrayView<ElementType>, false> {
     }
   }
 
+  /**
+   * @brief Constructs a property instance from a class definition and enum
+   * definition.
+   */
+  PropertyView(
+      const ClassProperty& classProperty,
+      const CesiumGltf::Enum* pEnumDefinition)
+      : _status(validateArrayPropertyType<PropertyArrayView<ElementType>>(
+            classProperty,
+            pEnumDefinition)),
+        _name(classProperty.name),
+        _semantic(classProperty.semantic),
+        _description(classProperty.description),
+        _count(_count = classProperty.count ? *classProperty.count : 0),
+        _offset(std::nullopt),
+        _scale(std::nullopt),
+        _max(std::nullopt),
+        _min(std::nullopt),
+        _required(classProperty.required),
+        _noData(std::nullopt),
+        _defaultValue(std::nullopt),
+        _propertyType(convertStringToPropertyType(classProperty.type)) {
+    if (_status != PropertyViewStatus::Valid) {
+      return;
+    }
+
+    if (classProperty.normalized) {
+      _status = PropertyViewStatus::ErrorNormalizationMismatch;
+      return;
+    }
+
+    getNumericPropertyValues(classProperty);
```

Comment:
`getNumericPropertyValues` handles `offset`, `scale`, `max`, `min`, which don't apply to enum properties. When you consolidate the constructors, just check the property type here.